README.md

```diff
@@ -59,6 +59,11 @@ OPTIONS:
         --texture1 <texture1>    Path to 2D RGBA texture for iChannel1
         --texture2 <texture2>    Path to 2D RGBA texture for iChannel2
         --texture3 <texture3>    Path to 2D RGBA texture for iChannel3
+        --wrap0 <wrap0>          Wrap mode for iChannel0 [default: "clamp"]
```

Comment:
Once you add the `possible_values` list to cli.yml as mentioned in the other comment, you can just copy-paste the output of the `--help` command here instead of manually editing it.

Reply:
Ah, I was wondering how that worked. Will do.